src/sa_web/static/js/models.js

```diff
@@ -290,6 +310,13 @@ var Shareabouts = Shareabouts || {};
   S.LandmarkModel = Backbone.Model.extend({
     initialize: function() {
       this.set("id", this.get('title'));
+    },
+    parse: function(response) {
+      var response = _.clone(response);
+      // add story object, if relevant
+      _.extend(response, addStoryObj(response.properties, "landmark"));
+
+      return response;
```

Comment:
Is this the correct way to handle parsing of the story object within `LandmarkModel`s?